pkg/kvstore/etcd.go

```diff
@@ -74,6 +77,9 @@ var (
 			EtcdOptionConfig: &backendOption{
 				description: "Path to etcd configuration file",
 			},
+			EtcdInitConnectionTimeout: &backendOption{
+				description: "Init Etcd connection timeout",
```

Comment:
`Etcd` should be all lowercase here. From what I can tell we use `etcd` consistently elsewhere.